distribution/docs/src/main/resources/_contents/_broker/configure-ha.adoc

```diff
@@ -31,17 +31,17 @@ config=file:etc/artemis-backup.xml
 If the ${branding} instances are currently running, stop and restart the backup and then the live server. Making sure the backup starts first ensures that the live server doesn't have any issues establishing the backup due to the backup being busy initializing. See <<Starting ${branding},Starting ${branding}>> for detailed steps.
 
 ==== Verify Cluster Replication
-Once both servers are up and running the following command can be run in curl or a https://localhost:8993/admin/jolokia/read/org.apache.activemq.artemis:type=Broker,brokerName=%22localhost%22,module=Core,serviceType=Server/ReplicaSync[browser] to verify that the servers have successfully synced.
+Once both servers are up and running the following command can be run in curl or a https://localhost:8993/admin/jolokia/read/org.apache.activemq.artemis:broker=%22localhost%22/ReplicaSync[browser] to verify that the servers have successfully synced.
```

Comment:
Again, I assume this is caused by a change in the way Artemis implemented their JMX?